content/github/setting-up-and-managing-your-github-profile/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Setting up and managing your GitHub profile
+intro: Learn how to customize your GitHub profile and manage your contribution graphs. 
```

Comment:
```suggestion
intro: You can customize your GitHub profile and manage your contribution graph. 
```
I modified this one just slightly to use the "You can" language that's in the example. We use that language to connect more directly with users. I also removed the `s` at the end of contribution graph, since users only have one graph in their profile.